config/settings.yml

```
@@ -44,6 +44,12 @@ solr:
     rule:
     shards:
     snitch:
+app_controller:
+  solr_and_fedora:
+    raise_on_connection_error: true
```

What do you think about setting this to false by default?  That would be a change in behavior but one that should only affect end users so I think it would be seen as an improvement.
```suggestion
    raise_on_connection_error: false
```